src/globals.ts

```diff
@@ -0,0 +1,29 @@
+import chokidar from 'chokidar';
+import chalk from 'chalk';
+import Command from '@oclif/command';
+
+export const GreenLog = chalk.hex('#00FF00');
+export const OrangeLog = chalk.hex('#FFA500');
+export const RedLog = chalk.hex('#FF0000');
+const CHOKIDAR_CONFIG = {
+  // awaitWriteFinish: true
```

Comment:
what about this one?

Reply:
I was bit confused whether to enable this or not. 
Usually specification files, having relatively larger size than our sample ones, may take some time to save.

When `awaitWriteFinish` is `true`
So the watcher triggers callback after 2 seconds, till the changes get saved. 
Also, that means, we dont get the log message instantaneously, wait for 2 seconds, after the change has been made.

Reply:
I have appended the explanation to the comment in the latest changes :)